packages/mobile/src/screens/track-screen/TrackRemixesScreen.tsx

```diff
@@ -47,20 +53,38 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
 
 export const TrackRemixesScreen = () => {
   const navigation = useNavigation()
-  const lineup = useSelectorWeb(getRemixesTracksLineup)
-  const count = useSelectorWeb(getCount)
-  const track = useSelectorWeb(getTrack)
-  const user = useSelectorWeb(getUser)
+  const lineup = useSelector(getRemixesTracksLineup)
+  const count = useSelector(getCount)
+  const track = useSelector(getTrack)
+  const user = useSelector(getUser)
+  const dispatch = useDispatch()
 
   const styles = useStyles()
+  const { params } = useRoute<'TrackRemixes'>()
+
+  const trackId = params.id
```

Comment:
The code I wrote assumes that `id` is always present + a valid track id - is that correct?

Reply:
I believe that's correct!